functional_hw/functional.py

```diff
@@ -0,0 +1,202 @@
+import sys
+
+
+def sequential_map(*args):
+    """
+    sequential_map takes several functions, and container of values to which these functions will be applied
+    and return list of results after application all functions to values container.
+
+    [!] Functions should return application results of them.
+    [!] Container of values should always be written last.
+    [!] Functions apply in order that they were written.
+
+    :param args: functions and container of values (last!).
+    :return: list with modified values from container.
+    """
+
+    *functions, values = args
+
+    for fun in functions:
+        values = list(map(fun, values))
+
+    return values
```

Comment:
Превратить элемент в список можно только в самом конце, map можно передавать в другие map'ы
```suggestion
    for fun in functions:
        values = map(fun, values)

    return list(values)
```